autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -85,8 +85,9 @@ def predict(self, X: Union[np.ndarray, pd.DataFrame],
     def estimator_supports_iterative_fit(self) -> bool:  # pylint: disable=R0201
         return False
 
-    def get_additional_run_info(self) -> None:  # pylint: disable=R0201
-        return None
+    def get_additional_run_info(self) -> Dict[str, Any]:  # pylint: disable=R0201
+        return {'pipeline_configuration': self.configuration,
+                'trainer_configuration': self.pipeline.named_steps['model_trainer'].choice.model.get_config()}
```

Comment:
It is not your fault, but I could not know what the `named_steps['model_trainer']` is.
If I understand codes correctly, it should be some models, right?
Could you put the reference to the file path (e.g. base class, typical examples of models) in the doc-string?

Reply:
sure

Reply:
BTW, you can read about named steps from scikit learn [here](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html).

Reply:
@franchuterivera Hum, but it does not help me...